managers/oclcCatalog.py

```diff
@@ -27,6 +35,47 @@ def queryCatalog(self):
                 classifyResp.status_code
             ))
             return None
-    
+
         return classifyResp.text
-        
\ No newline at end of file
+
+    def getToken(self):
+        try:
+            authResp = requests.post(
+                self.OCLC_AUTH_URL,
+                headers={ 'Content-Type': 'application/x-www-form-urlencoded'},
+                auth=(self.oclcClientID, self.oclcClientSecret),
+            )
+        except (Timeout, ConnectionError):
+            print(f'Failed to retrieve token from {self.OCLC_AUTH_URL}')
```

Comment:
Should we use the logger in favor of print statements, and if so should I go ahead and refactor the file to forward all of these to the logger?

Reply:
Hmm - ya I think let's add a logger to this class!